cli/cmd/public_api.go

```diff
@@ -1,5 +1,8 @@
 package cmd
 
+// TODO: this file should probably move out of `./cli/cmd` into something like
+// `./cli/publicapi` or `./cli/pkg`
```

Comment:
maybe create an issue and link to it in this TODO?

Reply:
filed #2735 yesterday to track, have updated the comment for reference.